src/Akeneo/Git/PullRequestCreator.php

```diff
@@ -68,8 +67,11 @@ public function __construct(
      * @param string      $baseDir
      * @param string      $projectDir
      * @param boolean     $dryRun
+     *
+     * @return array|null
+     * @throws \Exception
      */
-    public function create($baseBranch, $baseDir, $projectDir, $dryRun = false)
+    public function create($baseBranch, $baseDir, $projectDir, $dryRun = false): ?array
```

Comment:
not perfect, I hate returning `null` objects
this is clearly a bad thing to do

but if I don't want that I have to either:
- get rid of the dry run (is it really useful?)
- have 2 implementations of the PullRequestCreator (one for dry run, one for actual work)

but I won't start this refactoring in this PR